sam-app2/template.yaml

```diff
@@ -106,10 +111,21 @@ Globals:
     Layers:
     # Please see the README on the https://github.com/govuk-one-login/observability-infrastructure/blob/main/lambdalayer/README.md
     # , for the latest supported arn for your language.
-      - arn:aws:lambda:eu-west-2:216552277552:layer:Dynatrace_OneAgent_1_311_51_20250331-143707_with_collector_java:1
+      - !Ref OpenTelemetryLayerJava
 
 
 Resources:
+
+  OpenTelemetryLayerJava:
+    Type: AWS::Serverless::LayerVersion
+    Properties:
+      LayerName: JavaLayer
+      Description: Layer description
+      ContentUri: '../../../../aws-otel-java-agent-arm64-ver-1-32-0-339f86c3-4b25-4f5d-8adc-40717c26c118.zip'
```

Comment:
This could change to an S3 path, with a team only needing to maintain a version-controlled an immutable folder of zip files.